CHANGES.rst

```diff
@@ -20,6 +20,9 @@ New Features
     For JPL kernels, this roughly doubles the execution time, so if one requires
     only the positions, one should use ``get_body_barycentric``. [#5231]
 
+  - Transformations between coordinate systems can use the more accurate JPL
+    ephemerides. [#5273]
```

Comment:
Astrobot is going to complain that #5436 is not listed... Also same with the other entry.